tools/clang/lib/AST/ASTContextHLSL.cpp

```diff
@@ -698,6 +698,10 @@ void hlsl::AddRaytracingConstants(ASTContext &context) {
   AddConstUInt(
       context, StringRef("RAYTRACING_PIPELINE_FLAG_SKIP_PROCEDURAL_PRIMITIVES"),
       (unsigned)DXIL::RaytracingPipelineFlags::SkipProceduralPrimitives);
+  AddConstUInt(context, context.getTranslationUnitDecl(),
+               StringRef("RAYTRACING_PIPELINE_FLAG_ALLOW_OPACITY_MICROMAPS"),
+               (unsigned)DXIL::RaytracingPipelineFlags::AllowOpacityMicromaps,
+               ConstructAvailabilityAttribute(context, VT69));
```

Comment:
You will want a test that checks the availability attribute.  However, you'll need some kind of change to ensure that we pick it up.  Normally, we collect all the initializers for globals that could be used by any entry points.  Subobjects are not visible to entry points, so they would have to be collected independently to check availability attributes on constants like this one.  So I believe there will need to be additional work in that code to hook this up.